uportal-war/src/main/resources/layout/theme/respondr/navigation.xsl

```diff
@@ -55,6 +55,7 @@
 <xsl:param name="CONTEXT" select="'header'"/>
 <xsl:param name="subscriptionsSupported">true</xsl:param>
 <xsl:param name="USE_FLYOUT_MENUS" select="'false'" /> <!-- Sets the use of flyout menus.  Values are 'true' or 'false'. -->
+<xsl:param name="USE_OFFCANVAS_MENU" select="'true'" /> <!-- Sets the use of off-canvas menu.  Values are 'true' or 'false'. -->
```

Comment:
I'm not sure USE_FLYOUT_MENUS is actually a feature.  I think it was a placeholder for a feature that was on Universality and that we _intended_ to port to Respondr.